GPTResponder.py

```diff
@@ -17,17 +15,21 @@
 class GPTResponder:
     """Handles all interactions with openAI LLM / ChatGPT
     """
-    def __init__(self, convo: conversation.Conversation,
+    # By default we do not ping LLM to get data
+    enabled: bool = False
```

Comment:
should be self.enabled

Reply:
If this was inside the `__init__` method would need to use `self.enabled`. Declaration as a class variable does not need it.